### PR TITLE
V4: add bank_accounts endpoints

### PR DESCRIPTION
**Pull request by murilohns**
### Descrição

add bank_accounts endpoint

- [x] create
- [x] get a bank_accounts
- [x] get a list of bank_accounts

### CODE REVIEW

File: src/Endpoints/BankAccounts.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace PagarMe\Endpoints;
+
+use PagarMe\Client;
+use PagarMe\Routes;
+use PagarMe\Endpoints\EndpointInterface;
```
Comment (devdrops) [inline src/Endpoints/BankAccounts.php]: You don't need to declare this interface here, as `\PagarMe\Endpoints\Endpoint` already implements it :wink:

File: src/Routes.php [hunk 2]
```diff
@@ -43,4 +43,19 @@ public static function customers()
 
         return $anonymous;
     }
+
+    public static function bankAccounts()
```
Comment (devdrops) [inline src/Routes.php]: Missing docblock

Comment (murilohns) [reply src/Routes.php]: No one of these functions have docblock, do you want me to implement that?

Comment (devdrops) [reply src/Routes.php]: I see. And I've did that at #285, so, if you could only add the docblock to `bankAccounts` it woill help us a lot :smile:

**Comment by murilohns** [comment]
Done @devdrops

File: tests/unit/BankAccountsTest.php [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    {
+        $requestsContainer = [];
+        $handler = self::buildHandler($requestsContainer, $mock);
+
+        $client = new Client('apiKey', ['handler' => $handler]);
+
+        $response = $client->bankAccounts()->get(['id' => 1]);
+
+        $this->assertEquals('/1/bank_accounts/1', self::getRequestUri($requestsContainer));
+        $this->assertEquals('GET', self::getRequestMethod($requestsContainer));
```
Comment (leonampd) [inline tests/unit/BankAccountsTest.php]: Aqui eu usaria a constante definida em `Endpoint`:

https://github.com/pagarme/pagarme-php/blob/2a7b8a617f4ef4d8882dbf76f99f2028f52a7a01/src/Endpoints/Endpoint.php#L13-L16

Comment (leonampd) [reply tests/unit/BankAccountsTest.php]: O mesmo nas ocorrências acima.

Comment (devdrops) [reply tests/unit/BankAccountsTest.php]: Também pode usar `BanckAccounts::GET` que o resultado é o mesmo :wink:

File: tests/unit/RoutesTest.php [hunk 4]
```diff
@@ -54,4 +54,14 @@ public function testCardRoutes()
         $this->assertObjectHasAttribute('details', $routes);
         $this->assertIsCallable($routes->details);
     }
+
+    public function testBankAccountRoutes()
```
Comment (devdrops) [inline tests/unit/RoutesTest.php]: Missing docblock

**Pull request merged**